[2024-11-05T17:13:03.689+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-05T17:13:03.861+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag.specifications_scraping_tasks.cpu_scrape_task manual__2024-11-05T17:13:02.967561+00:00 [queued]>
[2024-11-05T17:13:03.872+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag.specifications_scraping_tasks.cpu_scrape_task manual__2024-11-05T17:13:02.967561+00:00 [queued]>
[2024-11-05T17:13:03.872+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-05T17:13:03.889+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): specifications_scraping_tasks.cpu_scrape_task> on 2024-11-05 17:13:02.967561+00:00
[2024-11-05T17:13:03.894+0000] {standard_task_runner.py:72} INFO - Started process 1666 to run task
[2024-11-05T17:13:03.897+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'specifications_scraping_tasks.cpu_scrape_task', 'manual__2024-11-05T17:13:02.967561+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpe70b834f']
[2024-11-05T17:13:03.899+0000] {standard_task_runner.py:105} INFO - Job 2: Subtask specifications_scraping_tasks.cpu_scrape_task
[2024-11-05T17:13:03.959+0000] {task_command.py:467} INFO - Running <TaskInstance: test_dag.specifications_scraping_tasks.cpu_scrape_task manual__2024-11-05T17:13:02.967561+00:00 [running]> on host 45fe9341983d
[2024-11-05T17:13:04.039+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='veil' AIRFLOW_CTX_DAG_ID='test_dag' AIRFLOW_CTX_TASK_ID='specifications_scraping_tasks.cpu_scrape_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-05T17:13:02.967561+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-05T17:13:02.967561+00:00'
[2024-11-05T17:13:04.039+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-05T17:13:04.050+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-05T17:13:04.051+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /app/scraper && scrapy crawl cpu_spider -O ../temp/cpu_data.json']
[2024-11-05T17:13:04.058+0000] {subprocess.py:86} INFO - Output:
[2024-11-05T17:27:21.391+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-11-05T17:27:21.393+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-11-05T17:27:21.397+0000] {process_utils.py:132} INFO - Sending 15 to group 1666. PIDs of all processes in the group: [1681, 1684, 1699, 1843, 1870, 1941, 2053, 2095, 2100, 2154, 1666]
[2024-11-05T17:27:21.398+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 1666
[2024-11-05T17:27:21.398+0000] {taskinstance.py:3092} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-11-05T17:27:21.399+0000] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
[2024-11-05T17:27:21.405+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1870, status='terminated', started='17:13:05') (1870) terminated with exit code None
[2024-11-05T17:27:21.406+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1941, status='terminated', started='17:13:05') (1941) terminated with exit code None
[2024-11-05T17:27:21.417+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1699, status='terminated', started='17:13:04') (1699) terminated with exit code None
[2024-11-05T17:27:21.418+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=2100, status='terminated', started='17:13:06') (2100) terminated with exit code None
[2024-11-05T17:27:21.432+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-05T17:27:21.515+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=2053, status='terminated', started='17:13:06') (2053) terminated with exit code None
[2024-11-05T17:27:21.516+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1843, status='terminated', started='17:13:04') (1843) terminated with exit code None
[2024-11-05T17:27:21.613+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1666, status='terminated', exitcode=0, started='17:13:03') (1666) terminated with exit code 0
[2024-11-05T17:27:21.614+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=2154, status='terminated', started='17:13:06') (2154) terminated with exit code None
[2024-11-05T17:27:21.614+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=2095, status='terminated', started='17:13:06') (2095) terminated with exit code None
[2024-11-05T17:27:24.012+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2024-11-05T17:27:31.676+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1684, status='terminated', started='17:13:04') (1684) terminated with exit code None
[2024-11-05T17:27:31.774+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1681, status='terminated', started='17:13:03') (1681) terminated with exit code None
